elastalert/loaders.py

```diff
@@ -605,9 +605,10 @@ def get_import_rule(self, rule):
         expanded_imports = []
         for rule_import in rule_imports:
             if os.path.isabs(rule_import):
-                expanded_imports.append(rule_import)
+                import_path = rule_import
             else:
-                expanded_imports.append(os.path.join(os.path.dirname(rule['rule_file']), rule_import))
+                import_path = os.path.join(os.path.dirname(rule['rule_file']), rule_import)
+            expanded_imports.append(os.path.normpath(import_path))
```

Comment:
The `expanded_imports` end up as dictionary keys in `import_rules`.  Normalizing the paths, i.e. removing the `./` and `../` to make testing easier.